src/apps/loyalty/containers/login/index.tsx

```diff
@@ -92,11 +92,17 @@ class Login extends React.Component<LoginProps, LoginState> {
   }
 
   handleEmailChange (e) {
-    this.setState({email: e.target.value})
+    this.setState({
+      email: e.target.value,
+      error: "",
```

Comment:
It is true that they both are falsey, but I think that `null` better reflects that difference and the _lack_ of an error.